README.md

```diff
@@ -15,6 +15,57 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
   * Gluesync  - A metastore event listener to replay Hive metadata events in a Glue catalog.
   * Metastore authorization - A metastore pre-event listener to handle authorization using Ranger.
 
+## Variables
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| apiary_customer_accounts | aws account ids for clients of this metastore | list | - | yes |
+| apiary_database_name | Database name to create in RDS for Apiary | string | `apiary` | no |
+| apiary_domain_name | Apiary domain name for route 53 | string | `` | no |
+| apiary_log_bucket | bucket for apiary logs | string | - | yes |
+| apiary_log_prefix | prefix for apiary logs | string | `` | no |
+| apiary_managed_schemas | schema names from which s3 bucket names will be derived,corresponding s3 bucket will be named as apiary_instance-aws_account-aws_region-schema_name | list | `<list>` | no |
+| apiary_producer_iamroles | aws iam roles allowed write access to managed apiary s3 buckets | map | `<map>` | no |
+| apiary_rds_additional_sg | Comma-seperated string for additional security groups to attach to RDS | list | `<list>` | no |
+| apiary_s3_alarm_threshold | will trigger cloudwatch alarm if s3 is greater than this, default 1TB | string | `10000000000000` | no |
+| apiary_tags | Common tags that get put on all resources | map | - | yes |
+| aws_region | aws region | string | - | yes |
+| db_backup_retention | The days to retain backups for, for the rds metastore. | string | - | yes |
+| db_backup_window | preferred backup window for rds metastore database in UTC. | string | `02:00-03:00` | no |
+| db_instance_class | instance type for the rds metastore | string | - | yes |
+| db_instance_count | desired count of database cluster instances | string | `2` | no |
+| db_maintenance_window | preferred maintenance window for rds metastore database in UTC. | string | `wed:03:00-wed:04:00` | no |
+| disable_database_management | disable creating and dropping databases from hive cli | string | `` | no |
+| ecs_domain_name | Domain name to use for hosted zone created by ECS service discovery | string | `lcl` | no |
+| elb_timeout | idle timeout for apiary ELB | string | `1800` | no |
+| enable_data_events | enable managed buckets s3 event notifications | string | `` | no |
+| enable_gluesync | enable metadata sync from hive to glue catalog | string | `` | no |
```

Comment:
Link these through to the relevant sub-module in apiary-extensions - i.e. https://github.com/ExpediaInc/apiary-extensions/tree/master/apiary-gluesync-listener